thundra/integrations/botocore.py

```diff
@@ -196,44 +175,41 @@ def get_operation_name(self, wrapped, instance, args, kwargs):
             if arn != 'N/A':
                 self.topicName = arn.split(':')[-1]
             else:
-                self.topicName = Constants.AWS_SERVICE_REQUEST
+                self.topicName = constants.AWS_SERVICE_REQUEST
 
         return self.topicName
 
     def getRequestType(self, string):
-        if string in Constants.SNSRequestTypes:
-            return Constants.SNSRequestTypes[string]
-        return Constants.AWS_SERVICE_REQUEST
-
-    def inject_span_info(self, scope, wrapped, instance, args, kwargs, response,
-                         exception):
+        if string in constants.SNSRequestTypes:
+            return constants.SNSRequestTypes[string]
+        return constants.AWS_SERVICE_REQUEST
```

Comment:
Return empty string instead of `AWS_SERVICE_REQUEST` constant if the request type is not known